### ISSUE TITLE
Heart Disease Prediction using 7 algorithms

### ISSUE THREAD

**Issue by shraiyya** [kritikaparmar-programmer/HealthCheck#11]
- I want to add the trained model which is used to predict if users suffer from heart disease or not. 
- The  classification models which are used are:

1. Logistic Regression
2. Random Forest Classifier
3. SVM
4. Naive Bayes Classifier
5. Decision Tree Classifier
6. XGBoost
7. Artificial Neural Network using Keras

- I am a GSSOC'21 participant

**Comment by kritikaparmar-programmer** [comment]
@shraiyya Assigning this issue to you, also one thing make a pr when the contribution period starts. Happy Coding :)

**Comment by shraiyya** [comment]
Yes Sure!

**Comment by ashish-hacker** [comment]
Can I work on this too @kritikaparmar-programmer ? I believe dividing the work and doing will be a lot faster. 
I am GSSOC'21 participant too.

**Comment by shraiyya** [comment]
@ashish-hacker  hey i am already done with the code. I will make the PR in a day or two

**Comment by ashish-hacker** [comment]
Ok 👍🏻 @shraiyya

**Comment by kritikaparmar-programmer** [comment]
@shraiyya updates?

**Comment by shraiyya** [comment]
I m done with the first 6 algorithms. I am trying to improve my ANN accuracy. I will definitely  submit before Monday EOD.
